### PR TITLE
Escape menu

### PR DESCRIPTION
**Pull request by jver422**
made an escape menu, did file management. added a thanks to Limezu in the readme and links to the assets provided by them as well as their itch.io page

### CODE REVIEW

File: Scenes/escape menu/escape menu.gd [hunk 1]
```diff
@@ -0,0 +1,25 @@
+extends Control
+
+var is_paused = false
+
+func _unhandled_input(event):
+	if event.is_action_pressed("pause"):
+		self.is_paused = !is_paused
+		get_tree().paused = is_paused
```
Comment (pyrrho) [inline Scenes/escape menu/escape menu.gd]: The `self.` on line 7 is confusing me. Should we always be referencing with `self`?

```
self.is_paused = !self.is_paused
get_tree().paused = self.is_paused
```

**Review by pyrrho — APPROVED** [review]
Maybe a nit, but I try to never use spaces in file/directory names. e.g. instead of `Scenes/escape menu/escape menu.gd`, `Scenes/escape_menu/escape_menu.gd`. Probably not a huge deal here since we're mostly using the Godot editor, and the plz-no-spaces thing makes command line interactions easier.

File: Scenes/escape menu/escape menu.gd (hunk 1 above)
Comment (jver422) [reply Scenes/escape menu/escape menu.gd]: oh, I was trying to use setget but turns out how that works changed between godot3 and 4, using self was supposed to stop setget from doing some infinite recursion but that's no longer the case. After abandoning the setget thing I just didn't think to remove it.

image

Comment (jver422) [reply Scenes/escape menu/escape menu.gd]: I just removed them

**Comment by jver422** [comment]
> Maybe a nit, but I try to never use spaces in file/directory names. e.g. instead of `Scenes/escape menu/escape menu.gd`, `Scenes/escape_menu/escape_menu.gd`. Probably not a huge deal here since we're mostly using the Godot editor, and the plz-no-spaces thing makes command line interactions easier.

good point, I'll be sure to look for that in the future. I've updated the file names

**Pull request merged**